android/keystores/BUCK

```diff
@@ -1,8 +0,0 @@
-keystore(
```

Comment:
any specific reason to remove this file?

Reply:
The BUCK file was used to define the build targets and dependencies for the Android app in the BUCK build system. However, since version 0.59 of React Native, Facebook has switched to using Gradle as the build system for React Native apps on Android, that's why BUCK file is no longer needed.